yt_dlp/extractor/facebook.py

```diff
@@ -390,7 +390,11 @@ def extract_metadata(webpage):
                 k == 'media' and str(v['id']) == video_id and v['__typename'] == 'Video')), expected_type=dict)
             title = get_first(media, ('title', 'text'))
             description = get_first(media, ('creation_story', 'comet_sections', 'message', 'story', 'message', 'text'))
-            uploader_data = get_first(media, 'owner') or get_first(post, ('node', 'actors', ...)) or {}
+            uploader_data = (
+                get_first(post, (..., ..., lambda k, v: k == 'owner' and v.get('name')), expected_type=dict)
+                or get_first(media, 'owner')
+                or get_first(post, ('node', 'actors', ...))
+                or {})
```

Comment:
`..., ...` seems maybe too lax for the first traversal path? i.e. it could branch out and return the wrong dict instead of falling back to the other `get_first`s. Is all that branching necessary? Or can we be more specific?

If you think we need `..., ...`, suggestion:
```suggestion
            uploader_data = (
                get_first(post, (..., ..., lambda k, v: k == 'owner' and v['name']))
                or get_first(media, ('owner', {dict}))
                or get_first(post, ('node', 'actors', ..., {dict})) or {})
```

Reply:
True, I've changed `get_first(media, ('owner', {dict}))` to be preferred way and added `video` as second expected branch. First path is something like `tahoe_sidepane_renderer` which I don't think we should stick to.